.github/ISSUE_TEMPLATE/bugreport.yaml

```
@@ -0,0 +1,60 @@
+name: Bug Report
+description: File a bug report
+labels: [bug, P5]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to report a bug report in [Uyuni](https://www.uyuni-project.org). Before you proceed, please ensure that there is no [issue](https://github.com/uyuni-project/uyuni/issues) for the bug yet.
+  - type: textarea
+    attributes:
+      label: Problem description
+      description: Include the expected and actual behavior. The more details, the better!
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Walk us through a minimal test case.
+      value: |
+        1.
+        2.
+        3.
+        ...
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Uyuni version
+      placeholder: |
+        Paste output of `zypper info Uyuni-Server-release`
+      render: bash
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Uyuni proxy version (if used)
+      placeholder: |
+        Paste output of `zypper info Uyuni-Proxy-release`
+      render: bash
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Useful logs
+      description: Do you see anything in the logs under `/var/log/rhn/` that looks like it could be related?
+      placeholder: |
+        Paste log excerpt
+      render: bash
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Additional information
+      description: Is these anything else that could help us?
```

```suggestion
      description: Is there any additional information that could help us understanding or fixing the issue?
```
Started fixing just the typo, but ended adding some words in there.